requirements/dev.txt

```diff
@@ -0,0 +1,43 @@
+# This requirements file is formatted to make it easy to see what are the
+# direct requirements of our application, and what are dependencies of
+# those requirements.
+#
+# The intent is to make this big mess of a file more maintainable. When removing
+# a dependency, it should be easy to see exactly what else can be removed. This
+# way we avoid keeping things in here that we don't require anymore.
+#
+# When adding a new requirement, it should fall in one of two categories:
+#
+# 1. Top-level requirements with no dependencies.
+# 2. Top-level requirement, followed by its dependencies.
+#
+# The first group is straight-forward: all top-level dependencies (resources that
+# are used by the application directly) are sorted by alphabetical order.
+# Dependencies that do not come from pypi (eg. eggs from github) are listed at
+# the end of that list.
+#
+# The second group are should be added to requirements-constraints.txt and follow conventions
+# explained there.
+# ----------------------------------------------------------------------------------
+#
+# requirements/default.txt contains all dependencies required to run python in the production environment.
+-r default.txt
+-c constraints.txt
```

Comment:
The `-c` is covered by `default.txt`, I think

Reply:
Fixed, thanks :)